lng/german.lng.php

```diff
@@ -1579,3 +1579,8 @@
 $lng['serversettings']['default_vhostconf']['description'] = 'Der Inhalt dieses Feldes wird direkt in den IP/Port-vHost-Container übernommen. '.$lng['admin']['specialsettings_replacements'].'<br /><strong>ACHTUNG:</strong> Der Code wird nicht auf Fehler geprüft. Etwaige Fehler werden also auch übernommen. Der Webserver könnte nicht mehr starten!';
 $lng['serversettings']['default_vhostconf_domain']['description'] = 'Der Inhalt dieses Feldes wird direkt in jeden Domain-vHost-Container übernommen. '. $lng['admin']['specialsettings_replacements'].'<strong>ACHTUNG:</strong> Der Code wird nicht auf Fehler geprüft. Etwaige Fehler werden also auch übernommen. Der Webserver könnte nicht mehr starten!';
 $lng['admin']['mod_fcgid_umask']['title'] = 'Umask (Standard: 022)';
+
+// HHVM Patch
+$lng['serversettings']['mod_fcgid']['hhvm'] = 'Kunden können HHVM statt PHP verwenden';
+$lng['domains']['hhvm']['title'] = "HHVM verwenden";
+$lng['domains']['hhvm']['description'] = "Auswählen um HHVM statt PHP zu verwenden.";
```

Comment:
where are the language strings in english.lng.php - which is the default and fallback and should always be present